src/main/org/pantsbuild/jarjar/Main.java

```diff
@@ -90,7 +94,24 @@ public void process(File rulesFile, File inJar, File outJar) throws IOException
     List<PatternElement> rules = RulesFileParser.parse(rulesFile);
     boolean verbose = Boolean.getBoolean("verbose");
     boolean skipManifest = Boolean.getBoolean("skipManifest");
-    MainProcessor proc = new MainProcessor(rules, verbose, skipManifest);
+    String misplacedClassStrategyName = System.getProperty("misplacedClassStrategy");
+    MisplacedClassStrategy misplacedClassStrategy = null;
+    if (misplacedClassStrategyName != null && misplacedClassStrategyName.length() > 0) {
+      if (misplacedClassStrategyName.equalsIgnoreCase("fatal")) {
```

Comment:
Or maybe a `MisplacedClassStrategyFactory` to encapsulate the different types and keep all of this logic out of Main